src/Security/MemberAuthenticator/ChangePasswordHandler.php

```diff
@@ -77,7 +81,8 @@ public function doChangePassword(array $data)
         $member->write();
 
         if ($member->canLogIn()->isValid()) {
-            $member->logIn();
+            Injector::inst()->get(IdentityStore::class)
```

Comment:
I like the idea, but I'm not sure how this would go time-wise. Do you mind if I park it on the to-do list of "Nice to have if plausible to achieve" @tractorcow?